client/src/components/common/ColorPicker.jsx

```diff
@@ -9,7 +9,7 @@ import LegendShape from '../charts/LegendShape';
 const ColorPicker = ({ title, left = 0, top = 0, placement, style, hideArrow, ...rest }) => (
   <Popover
     left={left}
-    top={top}
+    top={0}
```

Comment:
This component is used for several types of charts (see https://github.com/akvo/akvo-lumen/search?q=ColorPicker&type=code) I would guess this change will be a bit off for some chart types, or was that taken into consideration and it all works fine?

Reply:
Okay, I would effect it, just for the simple bar chart and see

Reply:
I have made the change just for the Simple bar chart component

Reply:
Cool!